lib/vmdb/appliance.rb

```diff
@@ -20,6 +20,14 @@ def self.log_config_on_startup
       Vmdb::Appliance.log_diagnostics
     end
 
+    def self.PRODUCT_NAME
+      I18n.t("product.name")
+    end
+
+    def self.USER_AGENT
+      "#{self.PRODUCT_NAME}/#{self.VERSION}".freeze
```

Comment:
Cool...you don't need the `self.` bits.

Reply:
I can never remember if an when you need `self`